django/core/mail/message.py

```diff
@@ -248,6 +248,12 @@ def __init__(
                 else:
                     self.attach(*attachment)
         self.extra_headers = headers or {}
+        self.headers_with_one_occurrence = {
+            "To": self.to,
+            "Cc": self.cc,
+            "Bcc": self.bcc,
```

Comment:
OK, `Bcc` is a different story, we should not include it in headers (check out ticket-18582 and a related ticket-28598). Please remove it from this patch.